RELEASING.md

```diff
@@ -79,17 +86,9 @@
 
 #### Update the Style Guide
 
-1. In [github/styleguide](https://github.com/github/styleguide), update `primer` to your newly released version in your terminal:
-
-  `npm install primer@latest`
-
-2. Then  run: `script/update-primer-docs`.
-
-3. Commit changes, make PR, get it approved, merge! 🚀
-
-#### Update [primer.github.io](primer.github.io)
+1. In [github/styleguide](https://github.com/github/styleguide), update `primer` by following [these instructions](https://github.com/github/styleguide/#adding-new-content-from-primer).
```

Comment:
We have update the style guide in the ship checklist above too, could we combine?

Reply:
Yes, I think that makes sense 👍